src/EditorFeatures/CSharpTest/ImplementInterface/ImplementInterfaceTests.cs

```diff
@@ -4657,7 +4690,12 @@ public void M24(double s = double.MaxValue)
     {
         throw new System.NotImplementedException();
     }
-}");
+}",
+                Options = { AllOptionsOff },
+
+                // 🐛 one value is generated with 0U instead of 0
+                CodeActionValidationMode = CodeActionValidationMode.None,
```

Comment:
📝 There are a few cases where validation needed to be disabled for this code fix. I'll file a follow-up issue to review them.

Reply:
➡️ https://github.com/dotnet/roslyn/issues/54329